### PR TITLE
small updates in jordan.py

### PR DESCRIPTION
**Pull request by Midhun134**
Changed the sigmoid function to use the standard sigmoid formula. 
Renamed dsigmoid to sigmoid_derivative for clarity. 
Modified the initialization of weights and biases to use numpy's randn function for better initialization. 
Updated the feedforward and backpropagation methods to use the new weight and bias initialization. 
Added a train method to train the network using backpropagation. 
Added a compute_loss method to compute the total loss on the training data. 
Updated the example code to use the new JordanNetwork class and training method.

### CODE REVIEW

**Comment by rougier** [comment]
Thanks. That's a lot of changes. What is your motivation exactly?

**Comment by Midhun134** [comment]
no motivation actually just being disciplined and consistent.

**Comment by Midhun134** [comment]
hi, My name is midhun, what's your name?
